recipes/aim/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "aim" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 83df85c5d4eadf8f9aa4c3cbe89564584627e4f7c90606be256bd7e36f009345
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - aim=aim.cli.cli:cli_entry_point
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - googlesearch
+    - protobuf
+    - anytree
+    - pyrser
+    - psutil
+  run:
```

Comment:
The requirements upstream are much longer and seem to be pinned very strictly: https://github.com/aimhubio/aim/blob/7d31a0feeb2d958cbaf5ed34e410737168a28c3d/setup.py#L39

Reply:
gotcha, just added more requirements by using grayskull this time

Reply:
@xhochy might have to WIP this, looks like we might not have `pysqlite3` in the conda-forge and the specific version of  `flask-migrate 2.7.0` either

Reply:
@xhochy I was able to the pins + additional requirements